runtime/predicates/reconcile_requested_test.go

```diff
@@ -0,0 +1,98 @@
+package predicates
```

Comment:
Can you please add a copyright header? We do this for any Go file in our projects.

Reply:
Added.